test/cypress/integration/social-share.test.js

```diff
@@ -0,0 +1,56 @@
+const socialShareButton = '[data-test="socialShareButton"]'
+const socialShareLinks = '[data-test="socialShareLinks"]'
+const socialShareSnackbar = '[data-test="socialShareSnackbar"]'
+
+describe('Social share', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('Social share expands on click and collapse on click.', () => {
+    cy.get(socialShareButton).click()
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('be.visible')
+    })
+    cy.get(socialShareButton).click()
+    cy.get(socialShareLinks).children().each(link => {
+      cy.wrap(link).should('not.be.visible')
+    })
+  })
+
+  it('Social share expands on enter and collapse on enter.', () => {
+    cy.get(socialShareButton).focus()
+    cy.get(socialShareButton).trigger('click')
```

Comment:
I don't think triggering the `click` event is the same as triggering an event with the keyboard by pressing the Enter key. Can you check this in Cypress?

Reply:
There is no way to simply press the enter key, after searching on the internet I found that the event triggered on enter is simply a 'click' event.